spec/index.bs

```diff
@@ -2183,20 +2183,20 @@ dictionary GPUBindGroupLayoutEntry {
     required GPUShaderStageFlags visibility;
     required GPUBindingType type;
 
-    // Used for uniform buffer and storage buffer bindings.
-    boolean hasDynamicOffset = false;
+    // Used for uniform buffer and storage buffer bindings. Must be undefined if not relevant.
```

Comment:
nit: Better to be more specific since we have context, e.g.:
```suggestion
    // Used for uniform buffer and storage buffer bindings. Must be undefined for other binding types.
```

Reply:
Done!